provider/win_driver_utils.py

```diff
@@ -0,0 +1,57 @@
+"""
+windows driver utility functions.
+
+:copyright: Red Hat Inc.
+"""
+import logging
+
+from virttest import utils_misc
+from virttest.utils_windows import wmic
+
+
+QUERY_TIMEOUT = 360
+INSTALL_TIMEOUT = 360
+OPERATION_TIMEOUT = 120
+
+
+def _pnpdrv_info(session, name_pattern, props=None):
+    """Get the driver props eg: InfName"""
+    cmd = wmic.make_query("path win32_pnpsigneddriver",
+                          "DeviceName like '%s'" % name_pattern,
+                          props=props, get_swch=wmic.FMT_TYPE_LIST)
+    return wmic.parse_list(session.cmd(cmd, timeout=QUERY_TIMEOUT))
+
+
+def uninstall_driver(session, devcon_path, driver_name, device_name,
+                     device_hwid):
+    """
+    Uninstall driver.
+
+    :param session: The guest session object.
+    :param devcon_path: devcon.exe path.
+    :param driver_name: driver name.
+    :param device_name: device name.
+    :param device_hwid: device hardware id.
+    """
+    devcon_path = utils_misc.set_winutils_letter(session, devcon_path)
+    status, output = session.cmd_status_output("dir %s" % devcon_path,
+                                               timeout=OPERATION_TIMEOUT)
+    if status:
+        logging.error("Not found devcon.exe, details: %s", output)
```

Comment:
Here should `return False`, the case can not continue this way and should raise `test.error`, right?

Reply:
@vivianQizhu thanks  have update it, thanks
run one of the case to check change it righ firstt:
(1/1) Host_RHEL.m8.u4.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.win_virtio_driver_install_from_update.with_viorng.q35: PASS (351.94 s)